akka-stream/src/main/mima-filters/2.5.12.backwards.excludes

```diff
@@ -4,3 +4,10 @@ ProblemFilters.exclude[ReversedMissingMethodProblem]("akka.stream.scaladsl.FlowO
 # #24758 recreate already closed substreams 
 ProblemFilters.exclude[DirectMissingMethodProblem]("akka.stream.impl.fusing.GroupBy.this")
 ProblemFilters.exclude[ReversedMissingMethodProblem]("akka.stream.scaladsl.FlowOps.groupBy")
+
+# #25135 Decrease the uninitialized connection counter as late as possible
+ProblemFilters.exclude[DirectMissingMethodProblem]("akka.stream.impl.io.IncomingConnectionStage.this")
```

Comment:
Do we consider excluding `akka.stream.impl.*` too dangerous? OK in any case, good to double-check exactly what changed I guess.